runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -147,6 +147,8 @@ J9::AheadOfTimeCompile::findCorrectInlinedSiteIndex(void *constantPool, uintptr_
       }
 
    TR_ASSERT(matchFound, "Couldn't find this CP in inlined site list");
```

Comment:
This should be removed as `matchFound` is a possible circumstance now.

Reply:
removed the assert as suggested.